spec/mailers/user_mailer_spec.rb

```diff
@@ -4,14 +4,20 @@
 require 'mailers/shared_examples_for_an_email'
 
 describe UserMailer do
-  describe '#daily_change_update' do
-    subject(:mail) { described_class.daily_change_update(user, change_updates).deliver_now }
+  describe '#notify_company_needs' do
```

Comment:
Si je ne m’abuse c’est `update_match_notify` :)
```suggestion
  describe '#update_match_notify' do
```